src/vscode-bicep/schemas/bicepconfig.schema.json

```diff
@@ -442,6 +442,16 @@
                     }
                   ]
                 },
+                "use-recent-api-version": {
+                  "allOf": [
+                    {
+                      "description": "Use recent API versions. See https://aka.ms/bicep/linter/use-recent-api-version"
```

Comment:
nit: Use recent API versions -> Use recent API version to match the rule name?

Reply:
Funny, I actually just changed the rule id to plural to match most other rules.  Will change class name later.